tests/test_user/test_login.py

```diff
@@ -0,0 +1,57 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from django.contrib.auth.models import User
+
+
+class AccountsTestCase(TestCase):
+    def setUp(self):
+        user = {
+            'username': 'demo_user'
+        }
+        User.objects.create_user(
+            username=user['username'], password='demo@123')
```

Comment:
Give the username directly here. There is no point in creating a separate dictionary just to hold the username which you are gonna use only once.

Reply:
changed